apps/central-scan/frontend/src/navigation_screen.tsx

```diff
@@ -92,40 +120,15 @@ export function NavigationScreen({ children, title }: Props): JSX.Element {
         <Link to="/">
           <CentralScanAppLogo appName="VxCentralScan" />
         </Link>
-        {isSystemAdministratorAuth(auth) && (
-          <NavList>
-            <NavListItem>
-              <NavLink
-                to="/system-administrator-settings"
-                isActive={isActivePath('/system-administrator-settings')}
-              >
-                Settings
+        <NavList>
+          {navItems.map(({ label, routerPath }) => (
+            <NavListItem key={routerPath}>
+              <NavLink to={routerPath} isActive={isActivePath(routerPath)}>
+                {label}
               </NavLink>
             </NavListItem>
-            <NavListItem>
-              <NavLink
-                to="/hardware-diagnostics"
-                isActive={isActivePath('/hardware-diagnostics')}
-              >
-                Diagnostics
-              </NavLink>
-            </NavListItem>
-          </NavList>
-        )}
-        {isElectionManagerAuth(auth) && electionDefinition && (
-          <NavList>
-            <NavListItem>
-              <NavLink to="/scan" isActive={isActivePath('/scan')}>
-                Scan Ballots
-              </NavLink>
-            </NavListItem>
-            <NavListItem>
-              <NavLink to="/settings" isActive={isActivePath('/settings')}>
-                Settings
-              </NavLink>
-            </NavListItem>
-          </NavList>
-        )}
+          ))}
+        </NavList>
```

Comment:
With this change, my read is that in an unconfigured state with Election Manager auth, we now render an empty `NavList` rather than no `NavList` at all. Can you confirm that doesn't change appearance?

Reply:
![Screenshot 2024-09-18 at 11 29 25 AM](https://github.com/user-attachments/assets/005ea978-2701-41dc-a38a-b606627708ef)
![Screenshot 2024-09-18 at 11 30 28 AM](https://github.com/user-attachments/assets/14784622-ae08-469d-8874-72be63f42f9f)

Good call. Yes, the top screenshot is with the empty `NavList` and the second screenshot is with no `NavList`, and they look the same. This makes sense given that the NavList will just render an [empty div](https://github.com/votingworks/vxsuite/blob/0f5f0fe6c068a21fe66446d63618906de462c837/libs/ui/src/left_nav.tsx#L27)

Reply:
FYI I also just gave Nikhil a deeper walkthrough of the UIs and their states, including what it means to be in an unconfigured state with election manager auth. So he'll be posting updated screenshots of that specific state accordingly!

Reply:
Yes, thank you! The previous screenshots just showed the comparison of an empty `NavList` vs no `NavList` through manual code changes to create the two states.

I've verified that current `main` EM auth/unconfigured (screenshot 1) matches the updated behavior from this PR (screenshot 2)

Screenshot 1 - `main` branch for EM/unconfigured machine
<img width="986" alt="Screenshot 2024-09-18 at 2 30 59 PM" src="https://github.com/user-attachments/assets/cd8fd3d0-d06b-4f7d-be0e-1808526787ed">

Screenshot 1 - this branch for EM/unconfigured machine
<img width="930" alt="Screenshot 2024-09-18 at 2 15 56 PM" src="https://github.com/user-attachments/assets/1971559e-1f59-44fb-a19b-5594e1d9f2a7">